frontend/packages/cli/src/cli/index.ts

```diff
@@ -18,8 +18,8 @@ program.addCommand(erdCommand)
 erdCommand
   .command('build')
   .description('Build ERD html assets')
-  .option('--input <path>', 'Path to the .sql file')
-  .option('--format <format>', 'Format of the input file')
+  .option('--input <path|url>', 'Path or URL to the .sql file')
```

Comment:
Can I change it like this so it is not limited to sql?

```suggestion
  .option('--input <path|url>', 'Path or URL to the schema file')
```

Reply:
oh, I see. nice!